package.json

```diff
@@ -16,15 +16,16 @@
     "biome:check": "biome check --apply-unsafe .",
     "biome:ci": "biome ci .",
     "test": "vitest",
+    "typecheck": "yarn workspace @joshwooding/vite-plugin-react-docgen-typescript typecheck",
     "storybook": "yarn workspace playground storybook"
   },
   "devDependencies": {
     "@biomejs/biome": "1.2.2",
     "@changesets/cli": "^2.25.2",
     "@types/react": "^18.0.0",
     "react": "^18.0.0",
-    "typescript": "^4.3.0",
-    "unbuild": "^1.0.2",
+    "typescript": "^5.0.0",
```

Comment:
For typescript, since each minor is a breaking change, I usually recommend this semver pattern: `5.0.X`.  Or, in this case, might as well use `5.2.X`